src/lib/support/JniReferences.cpp

```diff
@@ -388,4 +388,31 @@ CHIP_ERROR JniReferences::GetObjectField(jobject objectToRead, const char * name
     return err;
 }
 
+CHIP_ERROR JniReferences::CharToStringUTF(const char * value, jobject &outStr)
```

Comment:
Why not just pass in the CharSpan instead of doing the copy into an std::string but then just passing the char *?

Reply:
I modified. Thanks.